browser/BUILD.gn

```diff
@@ -182,12 +186,17 @@ source_set("browser_process") {
     "//content/public/browser",
     "//content/public/common",
     "//extensions/buildflags",
+    "//mojo/public/cpp/bindings",
     "//services/metrics/public/cpp:metrics_cpp",
     "//services/network/public/cpp",
     "//third_party/blink/public/mojom:mojom_platform_headers",
     "//third_party/widevine/cdm:buildflags",
     "//ui/base",
   ]
+  if (is_component_build) {
```

Comment:
for my own education - why is it needed?

Reply:
IIRC there were problems on Android or on desktop on linking, I will create a follow up to check exactly
https://github.com/brave/brave-browser/issues/13524